### ISSUE TITLE
Disable override button in performance page is not working

### ISSUE THREAD

**Issue by glictevo** [PrestaShop/PrestaShop#14596]
**Describe the bug**
The button to disable overrides is not working properly.

**To Reproduce**
1. Install a fresh PrestaShop
2. Add an override, for example for classes/CMS.php, override the __construct() function and add an echo 'hello'; followed by a die();
3. Delete the class_index in /var/cache/prod/class_index.php
4. Put yes on the disable overrides in the parameters in the back office
5. Go to any CMS page and modify it, the override is executed.
6. Even if i delete all the cache afterward it is still executed

I hope this is enough to reproduce the bug.

**Additional information**
PrestaShop version: 1.7.6.0
PHP version: 5.6

---

In the new version of the upgrade module, we should check if overrides are well disabled (see #14781)
We should fix this issue before working on the upgrade module

**Comment by khouloudbelguith** [comment]
Hi @glictevo,

Thanks for your report.
I have the issue with PS1.7.5.2 & PS1.7.6.0.
https://drive.google.com/file/d/12XsF7JDWlAsYbBL9ptPYW25T0Yh_WOsK/view
I’ll add this to the debug roadmap so that it’s fixed. If you have already fixed it on your end or if you think you can do it, please do send us a pull request!
Related to #9965
Thanks!

**Comment by glictevo** [comment]
Hello,

I've made a few researches.
The bug started at 1.7.3.0 (migration of the AdminPerformance page to Symfony, I don't know if it is linked)

However, I tested in 1.7.2.5, overrides are deactivated when switching the button to "Yes", however when deleting the cache by hand or with the "Empty cache" button in AdminPerformance controller, when the cache is regenerated, the bug is present and overrides are not deactivated. Then if switching to "No", save, reswitch to "Yes", save, then everything is fine and overrides are deactivated.
Same for 1.7.1.2 and 1.7.0.6, I haven't tested any version below this one yet.

Hope this can help a little for the moment.
